apps/gallery/index.html

```diff
@@ -57,7 +57,6 @@
     <link rel="stylesheet" type="text/css" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css">
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/gaia-icons.css" />
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/bidi-helper.css" />
-    <script defer src="shared/elements/gaia-header/dist/gaia-header.js"></script>
```

Comment:
As per https://developer.mozilla.org/en-US/docs/Mozilla/Firefox_OS/Developing_Gaia/Build_System_Primer#Build_script_for_apps
See 'Build Script for Apps'  utils.copyToStage in build.js copies gallery from app directory to build_stage